spec/models/payment_providers/base_provider_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe PaymentProviders::BaseProvider, type: :model do
+  subject(:provider) { described_class.new(attributes) }
+
+  let(:attributes) do
```

Comment:
just a best practice thing, can we put the `let` with block after the one lines `let` ? or does the order matter?